package.json

```diff
@@ -59,22 +59,22 @@
     "chai": "^3.3.0",
     "chai-as-promised": "^5.2.0",
     "concurrently": "^3.4.0",
-    "eslint": "^3.19.0",
+    "eslint": "^4.18.2",
     "eslint-config-airbnb": "^15.0.1",
     "eslint-config-airbnb-base": "^11.2.0",
     "eslint-plugin-import": "^2.3.0",
     "eslint-plugin-jsx-a11y": "^5.0.3",
     "eslint-plugin-react": "^7.0.1",
     "firefox-profile": "^1.0.3",
-    "geckodriver": "^1.15.1",
+    "geckodriver": "^1.19.1",
     "istanbul": "^0.4.5",
     "karma": "^1.7.1",
     "karma-chrome-launcher": "^0.2.3",
-    "karma-coverage": "^1.1.1",
+    "karma-coverage": "^2.0.1",
```

Comment:
Are we sure this is not a breaking change?

Reply:
I have created a new ticket to review all of them https://tokbox.atlassian.net/browse/ECO-4576 :)